utils/tzif/Cargo.toml

```diff
@@ -9,6 +9,7 @@ version = "0.1.0"
 edition = "2021"
 readme = "README.md"
 license-file = "LICENSE"
+keywords = ["time-zone", "posix", "iana", "parse", "data"]
 
```

Comment:
Nit: I am getting the following warning. Do you know what the problem is, and can you fix it?

```
warning: /path/to/icu4x/experimental/tzif/Cargo.toml: unused manifest key: package.author
```

Reply:
Yeah that key is deprecated now I think

Reply:
Ah, this was just a mistake, it should be `authors`. 

Thank you.